hamocc/mo_sedshi.F90

```diff
@@ -77,7 +79,7 @@ subroutine sedshi(kpie,kpje,omask)
                  & +      oplfa*sedlay(i,j,k,issssil) &
                  & +      clafa*sedlay(i,j,k,issster)
             ! "full sediment has sedlo=1
-            wsed(i,j)=max(0.,(sedlo-1.)/(sedlo+1.e-10))
+            wsed(i,j)=max(0.,(sedlo-1.)/(abs(sedlo)+1.e-10))
```

Comment:
Was this a bug?

Reply:
It doesn't change anything in the 3D, but it fixed running the sediment in the 1D setup.